nkms/network/server.py

```diff
@@ -80,3 +90,111 @@ class NuCypherSeedOnlyDHTServer(NuCypherDHTServer):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.storage = SeedOnlyStorage()
+
+
+class ProxyRESTServer(object):
```

Comment:
:+1:  I imagine this with have some implementation similarity with the `Miner` class in the kms-eth codebase.